quickwit/quickwit-cluster/src/grpc_gossip.rs

```diff
@@ -159,9 +159,9 @@ async fn wait_for_gossip_candidates(
             .skip_while(|node_states| {
                 node_states.len() < MAX_GOSSIP_ROUNDS
```

Comment:
MAX_GOSSIP_ROUNDS 's name is not exactly aligned with what it does.

Reply:
Yeah, it no longer makes sense. Will fix.